common/config/rush/pnpm-lock.yaml

```diff
@@ -4850,6 +4852,10 @@ packages:
     engines: {node: '>=6'}
     dev: false
 
+  /jenkins-hash-lookup3/1.0.0:
```

Comment:
I ended up dropping this package and cloning the .NET implementation because the package was operating on strings but we want to operate on byte arrays instead.